app/src/main/java/com/infomaniak/drive/data/cache/FileController.kt

```diff
@@ -783,11 +783,11 @@ object FileController {
 
     fun getFolderActivities(folder: File, page: Int, userDrive: UserDrive? = null): Map<out Int, FileActivity> {
         return getRealmInstance(userDrive).use { realm ->
-            getFolderActivitiesRec(realm, folder, page, userDrive)
+            getFolderActivitiesRecursive(realm, folder, page, userDrive)
         }
     }
 
-    private fun getFolderActivitiesRec(
+    private fun getFolderActivitiesRecursive(
```

Comment:
Maybe we can add `tailrec` to this function since it is recursive for memory optimisation?